src/initramfs/cmd/init/pkg/mount/mount.go

```diff
@@ -315,7 +317,7 @@ func appendBlockDeviceWithLabel(b *[]*BlockDevice, value string) error {
 	return nil
 }
 
-func probeDevice(devname string) (*BlockDevice, error) {
+func ProbeDevice(devname string) (*BlockDevice, error) {
```

Comment:
Not necessarily against making this public, but I'm curious because I don't see it being used outside this package.

Reply:
Unintentional. Thought I'd go this route at first, but didnt pan out.